pkg/rbac/clusterrole_test.go

```diff
@@ -0,0 +1,354 @@
+package rbac
+
+import (
+	"fmt"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/rbac/v1"
+)
+
+func TestNewClusterRoleBuilder(t *testing.T) {
+	testCases := []struct {
+		name              string
+		nsName            string
+		rule              v1.PolicyRule
+		expectedErrorText string
+		client            bool
+	}{
+		{
+			name: "test",
```

Comment:
nsName not set. On purpose?

Reply:
clusterRole is not namespaces resource.

Reply:
Can you remove nsName from the inputs of the test?

Reply:
fixed